tldw_Server_API/app/core/AuthNZ/federation/provisioning_service.py

```
@@ -0,0 +1,683 @@
+        user_id: int | None,
+        mapped_claims: dict[str, Any],
+    ) -> tuple[set[int], set[int], set[str], list[str]]:
+        warnings: list[str] = []
+        orgs_repo = AuthnzOrgsTeamsRepo(db_pool=self.db_pool)
+
+        desired_org_ids = set(_coerce_distinct_ints(mapped_claims.get("derived_org_ids")))
+        desired_team_ids: set[int] = set()
+        for team_id in _coerce_distinct_ints(mapped_claims.get("derived_team_ids")):
+            try:
+                team = await orgs_repo.get_team(int(team_id))
+            except Exception as exc:  # pragma: no cover - defensive runtime hardening
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)